css/components/navigation/menu-footer/menu-footer.css

```diff
@@ -16,12 +16,18 @@
   list-style: none;
 }
 
-.menu-footer__link:link,
-.menu-footer__link:hover {
-  background-color: inherit;
-  color: #fff;
-  font-weight: bold;
+.menu-footer a {
```

Comment:
We are trying as much as possible to theme items by class names instead of element type. 

Perhaps this section can be modified to theme by `.menu-footer__link` and `.menu-footer__link:active, etc` rather than by `.menu-footer a`

Reply:
It has to be more generic otherwise being overwritten by base styles. Or an `important!` will need to be added, which is worse IMO.